dashboard2/settings/jptt.md

```diff
@@ -152,6 +152,10 @@ HTML:
 ```
 **Note:** The JSON Path tester tool is **not equipped** to display the nested use case.
 
+##### JSON
+
+For JSON elements, the specific `json` selector is also available.
```

Comment:
This should be expanded. Like _how_ does it affect JSON paths? How do I make sense of it? How do I use it?